src/contexts/Sidebar/SidebarActions.js

```diff
@@ -1 +1,14 @@
-//code to be inputted later
+export const EXPAND = "EXPAND";
+export const CLOSE = "CLOSE";
+export const CHANGE = "CHANGE";
+
+//Is this necessary? They're not really imported anywhere from what I can see.
```

Comment:
Yes, they are necessary, they will be used in implementation. They're not imported anywhere right now because we haven't started implementing the reducers in the codebase, but these are what will be passed to the dispatch functions. For example, `sidebarDispatch(actionChangeSidebar('read'))`